publish-site.sh

```diff
@@ -42,6 +42,12 @@ if [[ "$1" == "--docker" ]]; then
   docker run -ti --name=vitess_publish_site -v $PWD/vitess.io:/in vitess/publish-site bash -c \
     'cp -R /in /out && cd /out && bundle install && bundle exec jekyll build'
   docker cp vitess_publish_site:/out/. vitess.io/
+  # There are cases where docker cp copies the contents of "out" in a
```

Comment:
Why would it happen sometimes and not others? We should be able to fix the copy command to be deterministic somehow...